app/scripts/controllers/metametrics.js

```diff
@@ -669,6 +679,9 @@ export default class MetaMetricsController {
         locale: this.locale,
         chain_id: properties?.chain_id ?? this.chainId,
         environment_type: environmentType,
+        ///: BEGIN:ONLY_INCLUDE_IN(desktop)
+        desktop_paired: this.getDesktopEnabled(),
+        ///: END:ONLY_INCLUDE_IN
```

Comment:
desktop_paired seems more like a user trait then a property we want on *every event*. @worldlyjohn thoughts?

Reply:
Thanks for reviewing my PR :bow:
If more appropriate I can move it to user traits.